draft-ietf-dnsop-server-cookies.md

```diff
@@ -164,13 +164,17 @@ The previous example in Appendix A.2 of [@!RFC7873] is NOT RECOMMENDED.
 
 # Constructing a Client Cookie {#clientCookie}
 
-The Client Cookie is a nonce and should be treated as such.  For simplicity, it
-can be calculated from Server IP Address, and a secret known only to the
-Client. The Client Cookie SHOULD have at least 64-bits of entropy.  If a secure
-pseudorandom function (like [@!SipHash-2.4]) is used, there's no need to change
-Client secret often. It is reasonable to change the Client secret only if it has
-been compromised or after a relatively long period of time such as no longer
-than a year.
+The Client Cookie is a **cryptographic** nonce and should be treated as such.
```

Comment:
The ** were used only to emphasize the change in email, it doesn’t have to go to RFC.

Reply:
Haha, but it's markdown, so will be translated in bold.  I'll remove it though...